packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue

```diff
@@ -32,7 +32,7 @@ withDefaults(
             :open="open" />
           <span
             v-if="!open && itemCount"
-            class="bg-b-2 text-c-2 text-3xs inline-flex h-4 w-4 items-center justify-center rounded-full font-semibold">
+            class="bg-b-2 text-c-2 text-3xs inline-flex h-4 w-4 items-center justify-center rounded-full font-semibold border-1/2">
```

Comment:
For future I think you can just write `border` because our default width is now `0.5px`

Reply:
![Cursor-2024-12-17-21-32-28@2x](https://github.com/user-attachments/assets/735ef426-1b81-4f08-9c90-e070ba5aca96)
![Cursor-2024-12-17-21-32-11@2x](https://github.com/user-attachments/assets/061c8c62-61a5-4daf-96c0-301a3add1b6a)

Reply:
Unless you wanted a `0.25px` border (which it might not do because rounding).